### DISCUSSION TITLE
zfs 2.1 releases

### DISCUSSION THREAD

**Discussion by jcb55** [openzfs/zfs#12133] [Q&A]
I see there are lot's of 2.1-rc releases and there is also a tag  2.1.99 
Are there some release notes for 2.1.99 ?

the rc releases do not seem stable, however the 2.1.99 at first look seems stable

**Comment by rincebrain** [comment]
The 2.1.99 tag is just a "fix" for #11916, it's not any more stable than the 2.1-rc releases.
